### PR TITLE
simulator: fixes from reviewing with dan

### PR DESCRIPTION
**Pull request by jeffkaufman**

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -233,20 +233,24 @@ <h1>Sampling and Sequencing Simulator</h1>
...
+or lower.
 <tr><td>Shedding duration (days):
     <td><input type=text id=shedding_duration value=5>
-    <td align=right>CV:<td><input type=text id=cv_shedding_duration>
+<td align=right>CV<sub>g</sub>:<td><input type=text id=cv_shedding_duration>
 <tr><td colspan=4 class=note>Once someone is infected, how many days do they
 shed for?  This is a simplified model where we assume people shed
-equally throughout their infection.
+equally throughout their infection. For CV<sub>g</sub> specify the
+geometric coefficient of variation.
```
Comment (dp-rice) [inline html/index.html:253]: Should be standard deviation? (Geometric mean = 0)

File: html/index.html [hunk 2]
```diff
@@ -353,39 +356,53 @@ <h1>Sampling and Sequencing Simulator</h1>
...
-  // the CV together.  Also do it in log space, since we want log
-  // normal here.
+  // Don't want to bias each mean independently, bias all of them
+  // together with log-normally distributed noise.  The geometric mean
+  // (and median) of the noise is zero, and the standard deviation is
+  // provided by the user.
 
-  const stdev_of_log = cv_of_log;
-  const bias_of_log = normal(1, stdev_of_log);
+  const bias = Math.exp(normal(0, geom_stdev));
```
Comment (dp-rice) [inline html/index.html:399]: It's equivalent, but you could use `lognormal(0, geom_stdev)` here to match above.

**Review by dp-rice — APPROVED** [review]
Left some minor comments, but substantively looks good!

File: html/index.html (hunk 1 above)
Comment (jeffkaufman) [reply html/index.html:253]: The code this describes is:

```
function get_input_geom_cv(input, input_geom_cv) {
  const geom_mean = get_input(input);
  const geom_cv = get_input(input_geom_cv) / 100;

  if (geom_cv < epsilon) {
    return geom_mean;
  }

  const geom_stdev = geom_cv * geom_mean;
  return lognormal(geom_mean, geom_stdev);
}
```

Unlike with $σ_g$ on the shedding amount, the mean that $CV_g$ applies to is the number of shedding days isn't zero.

File: html/index.html (hunk 2 above)
Comment (jeffkaufman) [reply html/index.html]: done

File: html/index.html (hunk 1 above)
Comment (jeffkaufman) [reply html/index.html]: Thanks for flagging this!  The function is lognormal(mu, sigma) and https://en.wikipedia.org/wiki/Log-normal_distribution says mu and sigma "are the expected value (or mean) and standard deviation of the variable's natural logarithm, not the expectation and standard deviation of [the variable] itself"

I think your correction is right if `mu = arithmatic_mean(log(X)) = log(geometric_mean(X))`.

Then for sigma I think it's easiest if I just have people specify it directly.  Done in ca63c42!

Comment (jeffkaufman) [reply html/index.html]: Also, stop using percentages, now that it's sigma and not CV: a52fa1d

**Pull request merged**